win32ss/user/ntuser/timer.c

```diff
@@ -19,7 +19,7 @@ static LONG TimeLast = 0;
 /* Windows 2000 has room for 32768 window-less timers */
 /* These values give timer IDs [256,32767], same as on Windows */
 #define NUM_WINDOW_MAX_TIMERS   (32768 - 1)
-#define NUM_WINDOW_LESS_TIMERS  (32768 - 256)
+#define MAX_WINDOW_LESS_TIMER_ID  (32768 - 256)
```

Comment:
It seems you renamed another define, not the one that was requested to rename by Timo 🤔

Reply:
yeah

Reply:
Sorry, I didn't realize it was about the first variable :) I've edited it.